keps/sig-api-machinery/3962-mutating-admission-policies/README.md

```diff
@@ -1032,8 +1032,8 @@ We will test the edge cases mostly in integration test and unit test. We may add
 
 #### Beta
 
-- Gather feedback from developers and surveys
-- Complete features A, B, C
+- Add metrics for MAP admission plugin
```

Comment:
Just checking.
https://github.com/kubernetes/enhancements/blob/893ac352d7954e1410d66fce872523e05c6eb7fe/keps/sig-api-machinery/3962-mutating-admission-policies/README.md#metrics

Are these metrics the ones you still want to implement?

Reply:
Yes mostly^^